linode_api4/objects/domain.py

```diff
@@ -61,3 +63,25 @@ def record_create(self, record_type, **kwargs):
 
         zr = DomainRecord(self._client, result["id"], self.id, result)
         return zr
+
+    def zone_file_view(self):
+        result = self._client.get(
+            "{}/zone-file".format(self.api_endpoint), model=self
+        )
+
+        return result["zone_file"]
+
+    def domain_clone(self, domain: str):
+        params = {"domain": domain}
+
+        self._client.post(
+            "{}/clone".format(self.api_endpoint), model=self, data=params
+        )
+
+    def domain_import(self, domain, remote_nameserver):
```

Comment:
`domain_import` -> `import`

Reply:
Since import is a reserved keyword I can't call the function import. Any other suggestions for what to call it?